package.json

```diff
@@ -104,7 +104,7 @@
   "eslintConfig": {
     "root": true,
     "parserOptions": {
-      "ecmaVersion": 5
+      "ecmaVersion": 6
```

Comment:
almost everything is ES5 _except_ [tools/uw-frame-static/build.js](https://github.com/UW-Madison-DoIT/uw-frame/blob/master/tools/uw-frame-static/build.js)